frontend/packages/operator-lifecycle-manager/src/components/operand/create-operand.tsx

```diff
@@ -121,110 +115,88 @@ export const CreateOperand: React.FC<CreateOperandProps> = ({
-                      csv.metadata.name,
-                      csv.metadata.namespace,
-                    ),
-                  },
-                  {
-                    name: t('olm~Create {{item}}', { item: model.label }),
-                    path: window.location.pathname,
-                  },
-                ]}
-              />
-            </div>
-            <PageHeading
-              badge={getBadgeFromType(model.badge)}
-              className="olm-create-operand__page-heading"
-              title={t('olm~Create {{item}}', { item: model.label })}
-            >
-              <span className="help-block">{helpText}</span>
-            </PageHeading>
-          </div>
-          <SyncedEditor
-            context={{
-              formContext: { csv, match, model, next, schema, providedAPI },
-              yamlContext: { next, match },
-            }}
-            FormEditor={FormComponent}
-            initialData={sample}
-            initialType={initialEditorType}
-            onChangeEditorType={onChangeEditorType}
-            prune={pruneFunc}
-            YAMLEditor={OperandYAML}
-          />
-        </>
-      ) : null}
-    </StatusBox>
+    <>
+      <div className="co-create-operand__header">
+        <PageHeading
+          badge={getBadgeFromType(model.badge)}
+          className="olm-create-operand__page-heading"
+          title={t('olm~Create {{item}}', { item: model.label })}
+        >
+          <span className="help-block">{helpText}</span>
+        </PageHeading>
+      </div>
+      <SyncedEditor
+        context={{
+          formContext: { csv, match, model, next, schema, providedAPI },
+          yamlContext: { next, match },
+        }}
+        FormEditor={FormComponent}
+        initialData={sample}
+        initialType={initialEditorType}
+        onChangeEditorType={onChangeEditorType}
+        prune={pruneFunc}
+        YAMLEditor={OperandYAML}
+      />
+    </>
   );
 };
 
-const stateToProps = (state: RootState, props: Omit<CreateOperandPageProps, 'model'>) => ({
-  model: state.k8s.getIn(['RESOURCES', 'models', props.match.params.plural]) as K8sKind,
-});
-
-export const CreateOperandPage = connect(stateToProps)((props: CreateOperandPageProps) => {
+const CreateOperandPage: React.FC<CreateOperandPageProps> = ({ match }) => {
   const { t } = useTranslation();
+  const [model] = useK8sModel(match.params.plural);
+  const [csv, loaded, loadError] = useK8sWatchResource<ClusterServiceVersionKind>({
+    kind: referenceForModel(ClusterServiceVersionModel),
+    name: match.params.csvName,
+    namespace: match.params.ns,
+    isList: false,
+  });
+
+  const DefaultPage = React.useCallback(
+    (props: { match: CreateOperandProps['match'] }) => (
+      <StatusBox loaded={loaded} loadError={loadError} data={csv}>
+        <CreateOperand {...props} csv={csv} initialEditorType={EditorType.Form} />
+      </StatusBox>
+    ),
+    [csv, loadError, loaded],
+  );
```

Comment:
Declaring a component inside of another component is an antipattern. It might be better to just load the extensions here separately.